src/ontoweaver/tabular.py

```diff
@@ -135,18 +135,21 @@ def make_id(self, type, entry_name):
         Raises:
             ValueError: If the ID creation fails.
         """
+        assert(type(entry_type) == str)
+        assert(type(entry_name) == str)
```

Comment:
Assert failing in case transformers are returning `int` values, for example the `rowIndex` transformer. I fixed this by turning all the yielded values from the transformers to `str`. Also in the docstring changed `type` to `entry_type`.